File: parantier-front/src/entities/work/types/workChecklist.ts
export interface WorkChecklist {
  id: number
  workId: number
  content: string
  checked: boolean
  imageUrl?: string
  imageFilename?: string
  orderNum: number
  createdAt: string
}

export interface CreateWorkChecklistRequest {
  content: string
  orderNum?: number
}

export interface UpdateWorkChecklistRequest {
  content: string
  checked: boolean
  orderNum: number
}

File: parantier-api/src/main/java/com/mapo/palantier/work/domain/WorkChecklist.java
package com.mapo.palantier.work.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WorkChecklist {
    private Long id;
    private Long workId;
    private String content;
    private Boolean isChecked;
    private String imageUrl;
    private String imageFilename;
    private Integer orderNum;
    private LocalDateTime createdAt;
}
